practice4/src/main/java/com/academiaDigital/softwareArchitecture/product/OrcCity.java

```diff
@@ -0,0 +1,8 @@
+package com.academiaDigital.softwareArchitecture.product;
+
+public class OrcCity implements IRaceCity{
+    @Override
+    public IRaceHouse[] getOrcHouses() {
+        return new IRaceHouse[0];

```

Comment:
what is the purpose of having this methods? is just for testing?

Reply:
According to the diagram we need to return an array, but I have changed in order to have a understandable test.